frappe/desk/page/activity/activity.py

```diff
@@ -38,7 +38,7 @@ def get_feed(start, page_length):
 				{match_conditions_comment}
 		) X
 		order by X.creation DESC
-		limit %(start)s, %(page_length)s"""
+		OFFSET %(start)s LIMIT %(page_length)s"""
```

Comment:
will this work with mysql too or its db dependent?

Reply:
@saurabh6790  Yes, it will work on both... It's Standard SQL